VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTextUtils.java

```diff
@@ -50,6 +50,12 @@ public static void setAuthorAttributionText(@NonNull TextView textView,
             final CharSequence formattedTime = getRelativeTimeSpanString(context,
                     initialTimeStampMs, authorData.getCreatedAt().getTime());
             final String authorLabel = authorData.getAuthorLabel();
+            final StringBuilder authorLabelFormatted = new StringBuilder();
+            if (authorLabel != null) {
+                authorLabelFormatted.append("(");
+                authorLabelFormatted.append(authorLabel);
+                authorLabelFormatted.append(")");
```

Comment:
It would be better to do this in the string resources themselves, instead of in code. This would also benefit localization.